tests/test1009.py

```diff
@@ -0,0 +1,45 @@
+# test some ligatures and fractions close to U0000. Avoid high values such as
+# UFxxx since the Unicode chart may add new unicode values later.
+
+import sys, fontforge
+
+print "Get Table Array Totals."
```

Comment:
Hmm why was the test skipped on Travis? Anyway adding brackets to the print calls should be enough for python 3 compatibility. `test1009.py` also needs to be added to `Makefile.am` `EXTRA_DIST` in the test folder.